### PR TITLE
fix(error): resolve error handler issues (DEV-938)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DEV-938

### CODE REVIEW

File: src/app/main/services/notification.service.ts [hunk 1]
```diff
@@ -17,16 +17,22 @@ export class NotificationService {
     // action: string = 'x', duration: number = 4200
     // and / or type: 'note' | 'warning' | 'error' | 'success'; which can be used for the panelClass
     openSnackBar(notification: string | ApiResponseError, type?: 'success' | 'error'): void {
-        let duration = 5000;
+        // let duration = 5000;
```
Comment (Vijeinath) [inline src/app/main/services/notification.service.ts:20]: Why not deleting this variable if it is not used?

Comment (kilchenmann) [reply src/app/main/services/notification.service.ts:20]: I thought I deleted it already... will do it asap

Comment (kilchenmann) [reply src/app/main/services/notification.service.ts]: done in 94c4c301

File: src/app/main/services/notification.service.ts [hunk 2]
```diff
@@ -17,16 +17,22 @@ export class NotificationService {
...
         let message: string;
-        let panelClass: string;
+        // let panelClass: string;
```
[the remark this answers is not in the document]
Comment (kilchenmann) [reply src/app/main/services/notification.service.ts]: done in 94c4c301

**Pull request merged**